src/python/pants/backend/terraform/dependency_inference_test.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Hm did you mean to have two separate test files? These look pretty similar. I think it's probably easiest to have one.

Reply:
That's what the pytest marker is for. The Pants tag says the file overall should run on macOS in CI, then we use Pytest to filter it down to only run the marked test and ignore the rest

Reply:
Combined them.